docs/sphinx_docs/Input_file_formats/index.rst

```diff
@@ -3,11 +3,17 @@
 Input file formats
 ******************
 
-Alt-ergo supports different input language. The main language is his native language, based on the lamguage of the Why plateform and detailed below. Alt-ergo (partially) supports the standard language of the SMT community, SMT-LIB. It also (partially) supports the input language of Why3 through the :doc:`AB-Why3 plugin <../Plugins/ab_why3>`.
+Alt-ergo supports different input languages:
+
+- The original input language is his native language, based on the language of the Why3 plateform and
+  detailed below.
+- Since 2.5.0, Alt-ergo fully supports the SMT-LIB language v2.6 through
+  `Dolmen <https://github.com/Gbury/dolmen>`_.
```

Comment:
```suggestion
- Alt-Ergo also supports the standard SMT-LIB language v2.6. Since version 2.5.0, improved support is provided by the `Dolmen <https://github.com/Gbury/dolmen>` frontend, available with `--frontend dolmen`.
```